src/OpenLoco/src/Paint/Paint.h

```diff
@@ -99,38 +101,29 @@ namespace OpenLoco::Paint
     };
     OPENLOCO_ENABLE_ENUM_OPERATORS(PaintStructFlags);
 
-#pragma pack(push, 1)
-    /* size 0x12 */
     struct AttachedPaintStruct
     {
-        ImageId imageId;        // 0x00
-        ImageId maskedImageId;  // 0x04
-        Ui::Point vpPos;        // 0x08
-        PaintStructFlags flags; // 0x0C
-        uint8_t pad_0D;
-        AttachedPaintStruct* next; // 0x0E
+        AttachedPaintStruct* next;
+        ImageId imageId;
+        ImageId maskedImageId;
+        Ui::Point vpPos;
+        PaintStructFlags flags;
     };
-    assert_struct_size(AttachedPaintStruct, 0x12);
 
     struct PaintStringStruct
     {
-        StringId stringId;       // 0x00
-        PaintStringStruct* next; // 0x02
-        Ui::Point vpPos;         // 0x06
-        uint16_t args[8];        // 0x0A
-        const int8_t* yOffsets;  // 0x1A
-        uint16_t colour;         // 0x1E
+        PaintStringStruct* next;
+        const int8_t* yOffsets;
+        uint16_t args[8];
+        Ui::Point vpPos;
+        StringId stringId;
+        uint16_t colour;
     };
-    assert_struct_size(PaintStringStruct, 0x20);
 
     struct PaintStructBoundBox
     {
-        int16_t x;    // 0x08
-        int16_t y;    // 0x0A
-        int16_t z;    // 0x0C
-        int16_t zEnd; // 0x0E NOTE: This order is important!
```

Comment:
is this order still important? or does your refactor change that?

Reply:
Now that interop has been removed, the order is no longer important.